src/components/AppBar/index.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '@material/tab/mixins';
+@import '@material/tab-indicator/mixins';
+
+.AppBar {
+  padding: 0 24px;
+
+  .title-grid {
+    align-items: center;
+    align-self: flex-end;
+    display: grid;
```

Comment:
Do we really need grid for this one? I kinda understand that it makes layout a little bit easier, but `grid-area` is [not supported in IE](https://caniuse.com/#feat=mdn-css_properties_grid-area). TBH, I think the same layout can be easily achieved using flexbox or even inline blocks, but correct me if I'm wrong.

Reply:
Fair, but are we supporting IE? Last I checked we only support 'last 2 versions' of evergreen browsers. Also, argument for grids, we'd have to add some hacky padding to the subtitle to get the text to line-up or rely on a pseudo-element for the logo in the main title.

Reply:
I think I have a non-hacky non-grid way of doing this but let's not let this block the PR -- let's merge this as-is and we can figure it out later if we have time.

For supported browsers, right now it's configured like this https://github.com/firebase/firebase-tools-ui/blob/486e7146f162663aa74a20797cad8878fdd8f65b/package.json#L41-L45 which I think includes IE11 (it's not "dead" yet). Whether we want that is a different question that we can talk for sure.

Reply:
How about this? https://github.com/firebase/firebase-tools-ui/pull/9